taskcluster/test-training-unittests_8k-linux-amd64-py35m-opt.yml

```diff
@@ -0,0 +1,12 @@
+build:
+    template_file: test-linux-opt-base.tyml
+    dependencies:
+      - "linux-amd64-ctc-opt"
+    system_setup:
+      >
+        apt-get -qq update && apt-get -qq -y install ${training.packages_trusty.apt}
+    args:
+      tests_cmdline: "${system.homedir.linux}/DeepSpeech/ds/taskcluster/tc-train-unittests.sh 3.5.8:m"
+    metadata:
+      name: "DeepSpeech on Linux AMD64 CPU training unittests using Python 3.5"
+      description: "Training a DeepSpeech LDC93S1 model for Linux/AMD64 Python 3.5, CPU only, optimized version"
```

Comment:
Can you also update the descriptions?

Reply:
What should I write there?

Reply:
How about "Training unit tests for Linux/AMD64 Python 3.5, CPU only, optimized version" (with version adjusted for the other two, of course)